mopidy_tidal/library.py

```diff
@@ -157,6 +147,13 @@ def search(self, query=None, uris=None, exact=False):
             logger.info("EX")
             logger.info("%r", ex)
 
+    @staticmethod
+    def _get_image_uri(obj):
+        try:
+            return obj.image
```

Comment:
Consider changing this to the below, so we can override the max resolution. Otherwise we will return really large images.
```
@staticmethod
def _get_image_uri(obj):
    try:
        return obj.picture(width=320, height=320)
    except AttributeError:
        pass
```

Reply:
Good one - I think it makes sense to limit the image resolution.

Reply:
@BlackLight Perhaps we should reconsider changing the resolution, as the images are almost too small when browsing Iris. Clicking on the larger preview results in a tiny image.

It looks like all types support 750x750 (I did some tests with the tidal web player and images using this resolution exists for playlists, albums and artists, even though this is not mentioned in the tidalapi source comments).

Reply:
@tehkillerbee I have added 750x750 as a standard resolution. 512x512 would have probably been a better trade-off, but in some of my tests several items didn't have 512x512 images associated.